packages/kubernetes/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.40.1-beta"
```

I would suggest to use `beta.1`, 1.40.0 hasn't been released.
```suggestion
- version: "1.40.0-beta.1"
```